segmentstore/server/src/test/java/io/pravega/segmentstore/server/writer/StorageWriterTests.java

```diff
@@ -747,6 +746,14 @@ public void testForceFlush() throws Exception {
         Assert.assertFalse("Not expected anything to be flushed the second time.", result2);
     }
 
+    /**
+     * Verifies that the StorageWriter does not create an Attribute Aggregator for Transient Segments.
+     */
+    @Test
+    public void testNoAttributeAggregatorTransientSegments() {
+
```

Comment:
Were you planning to write a test here?

Reply:
I did attempt to implement this test but wasn't really happy with the approaches I tried.. 

1. This first approach was to validate this by checking for side effects. I.E. add some Attributes during segment creation and check that core attributes are added to the attribute segment but extended attributes are not. The issue with this was it would take (potentially?) significant modifications to the `TestWriterDataSource` as the attribute segments are not created at the moment.
2. The second approach was to just directly check that there is no object of class `AttributeAggregator` in the `ProcessorCollection`. The problem with this was it requires changing the visibility of certain fields. Wasn't sure if its okay just to add a `VisibileForTesting` annotation.

So I just left it empty to cycle back to it if I thought of something better.